test/spec/validations/ambiguous_crossing_tags.js

```diff
@@ -139,17 +154,17 @@ describe('validationAmbiguousCrossingTags', () => {
       { 'crossing:markings': 'lines' }
     );
     const issues = validate();
-    verifySingleCrossingIssue(issues);
+    verifySingleCrossingWarning(issues);
   });
 
   it('flags an marked line and informal ladder node', () => {
     createWaysWithOneCrossingNode(
       { crossing: 'marked', highway: 'footway', footway: 'crossing'},
       { highway: 'residential' },
-      { 'crossing:markings': 'ladder', 'crossing':'informal'}
+      { 'highway':'crossing', 'crossing':'informal'}
```

Comment:
Does there need to be any mention of a ladder node here? Such as `'crossing:markings' : 'ladder'`?

Reply:
Ah, I didn't see the code diff while I was writing this.